CONTRIBUTING.md

```diff
@@ -61,7 +61,7 @@ The core implementation of this charm is represented by the [`GrafanaAgentOperat
 
 - `self.on.install`: In this event we patch K8s service ports.
 - `self.on.agent_pebble_ready`: In this event the charm builds the Pebble layer to be added in the workload charm. This Pebble layer will manage the execution of Promtail binary
-- `self.on["prometheus-remote-write"].relation_changed`: In this event the Grafana agent config is updated with remote-write settings.
+- `self.on["send-remote-write"].relation_changed`: In this event the Grafana agent config is updated with remote-write settings.
```

Comment:
```suggestion
- `self.on.send_remote_write_relation_changed`: In this event the Grafana agent config is updated with remote-write settings.
```

(This isn't actually important unless you care about it for taste, but I think accepting the suggestion also keeps the +1)